### ISSUE TITLE
[react-jss] Theming type is not exported

### ISSUE THREAD

**Issue by dfernandez-asapp** [cssinjs/jss#1309]
__Expected behavior:__

Since the function `createTheming` is exported by `react-jss`, I'll expect to import the `Theming` type from the same place:

```
import { createTheming,Theming } from 'react-jss';

const result : Theming<MyCustomTheme> = createTheming<MyCustomTheme>(ThemeContext);
```

__Describe the bug:__

```
import {
createTheming,
Theming // Fails Theming is not exported
} from 'react-jss';
```

__Codesandbox link:__ https://codesandbox.io/s/vigorous-keller-fsw88?fontsize=14&hidenavigation=1&theme=dark

__Versions (please complete the following information):__
 - jss: 10.1.1
 - Browser [e.g. chrome, safari]: Chrome
 - OS [e.g. Windows, macOS]: macOS

**Comment by kof** [comment]
Will be released in 10.1.1 or higher

**Comment by kof** [comment]
Released in v10.2.0
